model_analyzer/triton/client/client.py

```diff
@@ -53,6 +58,7 @@ def wait_for_server_ready(self, num_retries, sleep_time=1):
                     time.sleep(sleep_time)
                     retries -= 1
             except Exception as e:
+                self._check_for_triton_log_errors(log_file)
```

Comment:
am i reading line 55 correctly that even after the server is ready we still wait one more time?

Reply:
also, what is it that throws into this except block? The self._client.is_server_ready()? Because your code is only in the except block, so if nothing is throwing an exception it will never run

Reply:
Good point. I've added to the block above as well. 

I believe that what throws us into the exception block is the fact that the server has an error (due to the bad argument), but I've also seen us go into this block for non-errors, which is why I think the code is written the way it is: counting down retry in both cases and not aborting right away.